### PR TITLE
Read keys from k8s secrets

### PR DESCRIPTION
**Pull request by aguerra**
:horse:

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/169)

### CODE REVIEW

File: handlers/auth.go [hunk 1]
```diff
@@ -18,24 +18,38 @@ import (
...
 )
 
+func getKeyBytes(keyName string) ([]byte, error) {
+	s, err := k8s.Client.Secrets().Get(namespace, secretName)
+	if err != nil {
+		return nil, err
+	}
+	data, ok := s.Data[keyName]
+	if !ok {
+		return nil, fmt.Errorf("Missing key %s", keyName)
```
Comment (drgarcia1986) [inline handlers/auth.go]: Maybe create a error variable is more idiomatic, like `var ErrMissingKey = errors.New(fmt.Sprintf("Missing key %s", keyName))`, What do you think?

Comment (aguerra) [inline handlers/auth.go]: Didn't know that, I thought the shorter form was preferred. So:
```
func ....
    var errMissingKey = errors.New(fmt.Sprintf("Missing key %s", keyName))
    ...
    if !ok {
      return nil, errMisingKey
    } 
```
?

File: handlers/auth.go [hunk 2]
```diff
@@ -18,24 +18,40 @@ import (
...
+	secretName  = "teresa-keys"
+	namespace   = "kube-paas"
 )
 
 var (
 	verifyKey *rsa.PublicKey
 	signKey   *rsa.PrivateKey
 )
 
+var ErrMissingKey = std_errors.New("Missing key")
```
Comment (drgarcia1986) [inline handlers/auth.go]: What do you think about create it var inside previous var list ?

File: handlers/auth.go [hunk 3]
```diff
@@ -2,7 +2,7 @@ package handlers
 
 import (
 	"crypto/rsa"
-	"io/ioutil"
+	std_errors "errors"
```
Comment (drgarcia1986) [inline handlers/auth.go]: Why?

Comment (aguerra) [inline handlers/auth.go]: There's a `import "github.com/go-openapi/errors"` below.

File: handlers/auth.go (hunk 2 above)
Comment (aguerra) [inline handlers/auth.go]: :+1:

File: handlers/auth.go (hunk 3 above)
Comment (drgarcia1986) [inline handlers/auth.go]: 😢

**Pull request merged**